graylog2-server/src/main/java/org/graylog2/decorators/DecoratorImpl.java

```diff
@@ -81,7 +81,7 @@ public int compareTo(DecoratorImpl o) {
     public abstract Builder toBuilder();
 
     @JsonCreator
-    public static DecoratorImpl create(@JsonProperty(FIELD_ID) @JsonAlias("_" + FIELD_ID) @Id @ObjectId @Nullable String id,
```

Comment:
Was this added for backward compat? And we don't need it anymore?

Reply:
`@Id` transforms the `@JsonProperty("id")` into `@JsonProperty("_id")` internally anyway, so I think this is simply superfluous

Reply:
Okay :+1: